src/datarobotx/idp/custom_model_versions.py

```diff
@@ -86,7 +86,7 @@ def _get_or_create(
 
     dr.Client(token=token, endpoint=endpoint)  # type: ignore
     folder_path = kwargs.pop("folder_path", None)
-
+    max_wait = kwargs.pop("max_wait", 20 * 60)
```

Comment:
Any reason max_wait defaults to 30 mins while this one defaults to 20 mins?

Reply:
good catch